ci/tests/tracing/post.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+##TODO define create_keys() and exceed_limit() to setup AuthAPI and RateLimitAPI for traces
```

Comment:
No new todos in code

Reply:
It was a reminder for Monday Tomas :D working on it!